.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### :hammer_and_wrench: Description
+
+<!-- What code changed, and why? If an existing service SDK was updated, what functionality was added? If a new 
+version of a service SDK was added, what are the key new features or breaking changes? -->
```

Comment:
The [Kubernetes provider template](https://raw.githubusercontent.com/hashicorp/terraform-provider-kubernetes/master/.github/pull_request_template.md) includes a `Release note` section, which might be worth thinking about as a supplement to the description and PR title. I think for this project in particular, it can be hard to have meaningful release notes if the engineer cutting the release has to look back through the code changes.

Reply:
Oo good idea!